app/src/main/java/com/example/warriorsocial/ui/settings/SettingsFragment.java

```diff
@@ -37,6 +48,8 @@ public View onCreateView(@NonNull LayoutInflater inflater,
             // CHANGE THIS TO MATCH PRIVACY BUTTON WHEN ADDED
             btn_privacy = root.findViewById(R.id.btn_privacy);
 
+            changepass = root.findViewById(R.id.resetPass);
```

Comment:
I checked out/pulled from your branch and trying to run it, I got an error on this line.
Error implies that R.id.resetPass does not exist. For me, the change password button is listed as id "button4" on the settings_activity.xml
Changing line 51 to:
changepass = root.findViewById(R.id.button4);
fixed the problem for me.
Are you getting this same error?